### PR TITLE
Add examples to '-include-list' and '-exclude-list' flags

### LINKED ISSUE #2854: Add example for `include_list` in the `-config` section of `sct_run_batch`'s argparse help
It is unclear how to list subjects to include for the flag `include_list` when providing a yml/json file, so let's add it in the usage under flag `-config`:
~~~
include_list: ["sub-unf02", "sub-unf03"]
~~~

### LINKED ISSUE #4195: Make explicit that ses can be used in include/exclude params in `sct_run_batch`
### Description

In `sct_run_batch`, it can be useful to exlcude some sessions for example:

* ses-baselinebrain
* ses-baselinespinalcord

If you only want to process the spinal cord.

When including a session name in the argument `exclude-list`

~~~
-exclude-list [ ses-baselinebrain ses-followupbrain ]
~~~

These sessions are not processed.

E.g.:
~~~
sbedard@Mackeylab200:~/sc_analysis_test_2023-08-14-all/data_processed/sub-CR001$ ls
ses-baselinespinalcord  ses-followupspinalcord
~~~

This could be explicit in the `sct_run_batch ` args description that it can be done.

### PR DESCRIPTION
**Pull request by valosekj**
#### GitHub

- [x] I've given this PR a concise, self-descriptive, and meaningful title
- [ ] I've linked relevant issues in the PR body
- [x] I've applied [the relevant labels](https://intranet.neuro.polymtl.ca/geek-tips/contributing#pr-labels-a-href-pr-labels-id-pr-labels-a) to this PR
- [x] I've applied a [release milestone](https://github.com/spinalcordtoolbox/spinalcordtoolbox/milestones) (major, minor, patch) in line with [Semantic Versioning guidelines](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Misc%3A-Creating-a-new-release#convention-for-naming-releases) 
- [x] I've assigned a reviewer

#### PR contents

- [ ] I've consulted [SCT's internal developer documentation](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki) to ensure my contribution is in line with any relevant design decisions
- [ ] I've added [relevant tests](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Testing) for my contribution
- [ ] I've updated the [relevant documentation](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Documentation) for my changes, including argparse descriptions, docstrings, and ReadTheDocs tutorial pages

## Description

This is super minor PR, adding explicit examples to the `-include-list` and `-exclude-list` arguments.

## Related issues/PRs

Fixes #2854 
Fixes #4195

### CODE REVIEW

File: spinalcordtoolbox/scripts/sct_run_batch.py [hunk 1]
```diff
@@ -114,15 +114,17 @@ def get_parser():
...
+                        'Example: \'-include-list sub-001 sub-002\'', nargs='+')
     parser.add_argument('-exclude',
                         help='Optional regex used to filter the list of subject directories. Only process '
                         'a subject if they do not match the regex. Exclusions are processed '
                         'after inclusions. Cannot be used with `exclude-list`')
     parser.add_argument('-exclude-list',
                         help='Optional space separated list of subjects to exclude. Only process '
                         'a subject if they are not on this list. Inclusions are processed before exclusions. '
-                        'Cannot be used with either `exclude`.', nargs='+')
+                        'Cannot be used with either `exclude`. \n'
```
Comment (joshuacwnewton) [inline spinalcordtoolbox/scripts/sct_run_batch.py:126]: While we're here, we can fix this small typo too:

```suggestion
                        'Cannot be used with `exclude`. \n'
```

**Review by joshuacwnewton — COMMENTED** [review]
image

Looks good to me!

----

Though, I notice there is an open issue for this exact thing:

- https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/2854

And it seems to focus instead on the yaml usage (under `-config`). :thinking: 

----

I notice that there is also this related issue:

- https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4195

Perhaps we should tackle this one at the same time, too, since they're related? :thinking:

**Review by sandrinebedard — COMMENTED** [review]
Looks great! do we also want to add the same example in `-exclude-list` ? I know it is the same as `-include-list` but some people may not read it

**Comment by valosekj** [comment]
> Though, I notice there is an open issue for this exact thing:
> 
> * [Add example for `include_list` in the `-config` section of `sct_run_batch`'s argparse help #2854](https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/2854)
> 
> And it seems to focus instead on the yaml usage (under `-config`). 🤔

I believe the example of how to use `include_list`/ `exclude_list` in the YML config file was already addressed in the past, see line 67:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/00df6adcbe4db561c395d42c80d3433350b26d80/spinalcordtoolbox/scripts/sct_run_batch.py#L67

The confusion here is that the syntax is different between the config file and CLI.

config file - comma-separated list WITH quotation marks (`""`):

```console
["sub-01", "sub-02"]
```

CLI - space-separated list WITHOUT quotation marks:

```console
sub-001 sub-002
```

**Comment by valosekj** [comment]
> Looks great! do we also want to add the same example in `-exclude-list` ? I know it is the same as `-include-list` but some people may not read it

I added examples for both `include_list`/`exclude_list` in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4361/commits/00df6adcbe4db561c395d42c80d3433350b26d80

**Review by joshuacwnewton — APPROVED** [review]
Thanks for this! :hearts:

File: spinalcordtoolbox/scripts/sct_run_batch.py [hunk 2]
```diff
@@ -112,17 +112,19 @@ def get_parser():
                         'a subject if they match the regex. Inclusions are processed before exclusions. '
                         'Cannot be used with `include-list`.')
     parser.add_argument('-include-list',
-                        help='Optional space separated list of subjects to include. Only process '
-                        'a subject if they are on this list. Inclusions are processed before exclusions. '
-                        'Cannot be used with `include`.', nargs='+')
+                        help='Optional space separated list of subjects or sessions to include. Only process '
+                        'subjects or sessions if they are on this list. Inclusions are processed before exclusions. '
+                        'Cannot be used with `include`. \n'
+                        'Examples: \'-include-list sub-001 sub-002\' or \'-include-list ses-01 ses-02\'', nargs='+')
```
Comment (sandrinebedard) [inline spinalcordtoolbox/scripts/sct_run_batch.py:118]:
```suggestion
                        'Examples: \'-include-list sub-001 sub-002\' or \'-include-list sub-001/ses-01 ses-02\'', nargs='+')
```

Comment (sandrinebedard) [reply spinalcordtoolbox/scripts/sct_run_batch.py:118]: maybe to show how to use both together

**Review by sandrinebedard — APPROVED** [review]

**Comment by valosekj** [comment]
CI/CD has finished successfully --> merging

**Pull request merged**